bundlesize.config.json

```diff
@@ -34,7 +34,7 @@
     },
     {
       "path": "packages/autocomplete-theme-classic/dist/theme.min.css",
-      "maxSize": "4.25 kB"
+      "maxSize": "4.5 kB"
```

Comment:
The CSS size increased a little because of how the updated dependencies compile the Sass code. Ultimately, this should be more correct and resilient code anyway and I didn't see issues when doing visual comparisons (feel free to check the CodeSandbox demos).